File: Javascript/Fibonacci.js
function fibonacci(depth) {
  let lastIndex = 0;
  let newIndex = 1;
  let phi = [];

  phi[0] = lastIndex;
  phi[1] = newIndex;
  for (i = 2; i < depth; i++) {
    let result = lastIndex + newIndex;
    phi[i] = result;
    lastIndex = newIndex;
    newIndex = result;
  }
  console.log(phi);
}

fibonacci(10);

File: Python/Fibonacci.py
def fibonacci(depth):
    lastIndex = 0
    newIndex = 1
    phi = []

    phi.append(lastIndex)
    phi.append(newIndex)
    for i in range(2, depth):
        result = lastIndex + newIndex
        phi.append(result)
        lastIndex = newIndex
        newIndex = result
    print(phi)


fibonacci(10)
